pipe_vessels/elasticsearch/importer.py

```diff
@@ -0,0 +1,92 @@
+import json
+import sys
+import datetime as dt
+import itertools as it
+import json
+import re
+from pipe_vessels.elasticsearch.server import ElasticSearchServer
+
+
+def batch(iterable, size):
+    args = [iter(iterable)] * size
+    return it.izip_longest(*args)
+
+
+bq_datetime_regex = re.compile('^(\d\d\d\d-\d\d-\d\d) (\d\d:\d\d:\d\d) UTC$')
+
+
+def to_elasticsearch_datetime(value):
```

Comment:
@andres-arana I think we need to make a pre-requisite that data comes in with the needed format for timestamps.